api/PaymentRequest.json

```diff
@@ -15,12 +15,36 @@
             "version_added": "15"
           },
           "firefox": {
-            "version_added": false,
-            "notes": "Available only in nightly builds. Requires <code>dom.payments.request.enabled</code> to be set to <code>true</code> and the comma-delineated list in <code>dom.payments.request.supportedRegions</code> to contain one or more of the supported 2-character ISO locales, currently US and CA."
+            "version_added": "55",
+            "notes": "Available only in nightly builds.",
```

Comment:
This not appears to be wrong. The default value of dom.payments.request.enabled appears to be false in both Firefox Nightly (94) and Firefox 92. (If it were actually true, then we should use "preview" as the version number, not a note like this.)

I see this note is everywhere for payments stuff, can you either clean it all up or file an issue about it?

Reply:
I've converted this note into the appropriate flag data for all of the instances in both this API and `PaymentRequestUpdateEvent`.  I'll get `PaymentMethodChangeEvent` and `PaymentResponse` updated shortly!

Reply:
> I've converted this note into the appropriate flag data for all of the instances in both this API and `PaymentRequestUpdateEvent`.  I'll get `PaymentMethodChangeEvent` and `PaymentResponse` updated shortly!

- - - **_[]()_**